CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ description: |-
 # VKCS Provider's changelog
 
 #### v0.2.0 (unreleased)
+- Add datasources for DB datastore capabilities and configuration parameters
+- Add datasources for DB datastores
```

Comment:
resources
datasources
arguments/attributes
fixes

maybe we should set such order?